packages/lib-user/src/components/UserStats/components/ProfileHeader.js

```diff
@@ -0,0 +1,139 @@
+import { Box, Image } from 'grommet'
+import { number, string } from 'prop-types'
+import styled from 'styled-components'
+import { SpacedText, withResponsiveContext, ZooniverseLogo } from '@zooniverse/react-components'
+
+const StyledAvatar = styled(Image)`
+  width: 50px;
+  height: 50px;
+  object-fit: cover;
+  border-radius: 50%;
+`
+
+// TitledStat component will be moved to a shared folder when necessary
+
+function TitledStat ({
+  title = '',
+  value = 0 
+}) {
+  return (
+    <Box
+      align='center'
+    >
+      <SpacedText
+        color={{ dark: 'neutral-6', light: 'neutral-7' }}
+        size='xsmall'
+        uppercase={false}
+      >
+        {title}
+      </SpacedText>
+      <SpacedText
+        color='brand'
+        size='xlarge'
+        weight='bold'
+      >
+        {value}
+      </SpacedText>
+    </Box>
+  )
+}
+
+TitledStat.propTypes = {
+  title: string.isRequired,
+  value: number.isRequired
+}
+
+function ProfileHeader ({
+  avatar = '',
+  classifications = 0,
+  contributors = 0,
+  displayName = '',
+  hours = 0,
+  login = '',
+  projects = 0,
+  screenSize = 'large'
```

Comment:
I don't think we officially have a 'large' breakpoint defined in our Grommet theme yet - just 'small' and 'medium'. See https://github.com/zooniverse/front-end-monorepo/issues/4186. It's good you've used 'small' as the case to check against for certain styles, but `screenSize` should be initialized as 'medium' for now.